src/pages/TaskTriageDetailsPage.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+import { makeStyles } from '@material-ui/styles';
+import { useSelector } from 'react-redux';
+import {
+  Form, Helmet, useTranslations, useModulesManager, useHistory,
+} from '@openimis/fe-core';
+import TaskHeadPanel from '../components/TaskHeadPanel';
+import TaskPreviewPanel from '../components/TaskPreviewPanel';
+
+const useStyles = makeStyles((theme) => ({
+  page: theme.page,
+}));
+
+function TaskTriageDetailsPage() {
```

Comment:
I think that this component can be renamed for TaskDetailsPage. It's not exclusive for Triage user

Reply:
You're right. It has been changed.